### PR TITLE
Update Kext-Updater from 3.5.0 to 3.5.3

### PR DESCRIPTION
**Pull request by iMonZ**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ X] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/kext-updater.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 cask 'kext-updater' do
+
```
Comment (ran-dall) [inline Casks/kext-updater.rb:2]:
```suggestion
```

**Review by ran-dall — CHANGES_REQUESTED** [review]
@iMonZ The extra white line isn't needed.

**Comment by iMonZ** [comment]
Hey Thank you but I haven't added an extra line at the commit

**Comment by ran-dall** [comment]
> Hey Thank you but I haven't added an extra line at the commit

`C:  2:  1: Extra empty line detected at block body beginning.`

**Pull request merged**